### ISSUE TITLE
How do I combine hidden tracks if found

### ISSUE THREAD

**Issue by stephen147** [ghDaYuYu/foo_discogs#31]
With this [release ](https://www.discogs.com/release/11099319)I get this in the title of the 12th Hidden track _**Bonus Tracks (La La La (Excuse Me Miss Again))**_

Is it possbile to gather all the hidden track titles and place them in the title tag like below?

**_La La La (Excuse Me Miss Again), Stop, Beware... (Jay-Z Remix) and What They Gonna Do Part II_**

So the format would be like:
**_Track 1, Hidden Track 2 and Hidden Track 3_**

**Comment by gitgetgot** [comment]
try 

`$ifequal(%TRACK_TOTAL_HIDDEN_TRACKS%,0,%TRACK_TITLE%,$join($extend($array(%TRACK_TITLE%),%<TRACK_HIDDEN_TRACKS_TITLE>%),' / '))`

That will give you:

`Hidden Bonus Tracks (La La La (Excuse Me Miss Again)) / Hidden Bonus Tracks (Stop ) / Hidden Bonus Tracks (Beware... (Jay-Z Remix)) / Hidden Bonus Tracks (What They Gonna Do Part II)`

The easy part is to change the ' / ' with ', ' but I never figured out, how to strip/ignore all the repetitions of 'Hidden Bonus Tracks' in the '%<TRACK_HIDDEN_TRACKS_TITLE>%'.

But it is a start...

**Comment by ghuDaYuYu** [comment]
Just a brief update on this issue...
I guest that at some point this issue will be addressed, unfortunately not soon. The reasons being that there is previous work that should be previously done: a full revision/rewrite of big parts of the track parsing algorithms, including quite some effort testing the impact of those changes. The code revisions where drafted a while ago, but could not find the time to fully test them.
The outcome would be better hidden tracks management/detection, extended track header functionality (good for classical music),  improved disk#, track# and vinyl side parsing, etc. Unfortunately, I am still pretty unsure how this can be accomplished, maybe through I different branch of the project if there is enough man/hours available to attempt it.
Cheers!

**Comment by stephen147** [comment]
No worries. It's not a regular enough occurrence to bother me and I can manually do them when I find any. But it would be a nice thing to have fixed.

Thanks again for the hard work you're putting into this. Sorry, I can't be of much help :/

**Comment by ghuDaYuYu** [comment]
Yes, no worries. I might enable an optional algorithm choice, at some point, in the preferences, to be turn on for those stubborn releases/cd collections that refuse to be parsed by the original algorithm.

You're welcome, done a lot of collaborations already so thanks to you too.
Good seeing you!

**Comment by stephen147** [comment]
Good to see you too. Happy new year, even though a month has nearly passed 😁😁😁

That optional thing might be a good way to test it out. I'd be happy to test it for you.
